tasks/collect-mrrc-params/collect-mrrc-params.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-mrrc-params
+spec:
+  params:
+    - name: dataJsonPath
+      type: string
+      description: path to data json file
+  results:
+    - name: mrrcParamFilePath
+      description: path of the env file for following tasks
+    - name: cosignPubKeyConfig
+      description: the configmap name for cosign verify public key
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+  steps:
+    - name: collect-mrrc-params
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/usr/bin/env bash
+        set -eux
+
+        DATA_FILE="$(workspaces.data.path)/$(params.dataJsonPath)"
+
+        MRRC_ENV_FILE_PATH="$(dirname $DATA_FILE)/mrrc.env"
```

Comment:
This also should be in the workspace

Reply:
The way it's implemented now it **is** in the workspace (because it's derived from `$DATA_FILE`. But there is one issue - the result should contain the relative path under the workspace, not the absolute path.

Reply:
Ah, thanks for correcting me. Hopefully CI passes once this last part is fixed

Reply:
Um... I think the `dirname $DATA_FILE` will give the absolute path of the data.json, including both the workspace dir and the relative dir. Not sure why it can not work in test. But FWIW I've changed the code to use `dirname $(params.dataJsonPath)` here which just use the relative path now, which means this MRRC_ENV_FILE_PATH will just be a relative path in the workspace dir.